intake/__init__.py

```diff
@@ -41,7 +41,7 @@ def __getattr__(attr):
     """
     gl = globals()
     if attr in openers:
-        driver = openers[attr].load()
+        driver = registry[attr[5:]]  # "open_..."
```

Comment:
Might be worth checking that the first five chars really are `open_` to avoid confusing failure modes.